rllib/policy/sample_batch.py

```diff
@@ -1350,3 +1239,164 @@ def __repr__(self):
         return "MultiAgentBatch({}, env_steps={})".format(
             str(self.policy_batches), self.count
         )
+
+
+@PublicAPI
+def concat_samples(samples: List[SampleBatchType]) -> SampleBatchType:
+    """
```

Comment:
Same here: Please always put a single-line, single-sentence description in the very first line of the docstring.